src/APIFunctions/ErrorLog.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios' // what is this??
+
+export async function getAllErrorLogs () {
+  let allErrorLogs = []
+  await axios
+    .get('api/ErrorLog/getErrorLogs')
+    .then(res => {
+      allErrorLogs = res.data
+    })
+    .catch(err => {
+      allErrorLogs = err
+    })
+  return allErrorLogs
+}
+
+export async function AddErrorLog (newError) {
+  let errorCreated = false
+  const errorToAdd = {
+    userEmail: newError.userEmail,
+    errorTime: newError.errorTime,
+    apiEndpoint: newError.apiEndpoint,
+    errorDescription: newError.errorDescription
+  }
+  await axios
+    .post('/api/ErrorLog/addErrorLog', { ...errorToAdd })
```

Comment:
```suggestion
    .post('/api/ErrorLog/addErrorLog', { ...newError })
```

Reply:
the reason I changed it to errorToAdd because it is giving me the following error if I change it to newError 
![image](https://user-images.githubusercontent.com/29296452/75628675-13926180-5b90-11ea-8497-1be00260358e.png)

Reply:
This is because you no longer use the variable and need to remove it.